cmd/controller/controller.go

```diff
@@ -66,7 +67,11 @@ func NewController(clientset helmClientset.Interface, kubeClient kubernetes.Inte
 		UpdateFunc: func(oldObj, newObj interface{}) {
 			key, err := cache.MetaNamespaceKeyFunc(newObj)
 			if err == nil {
-				queue.Add(key)
+				newReleaseObj := newObj.(*helmCrdV1.HelmRelease)
+				oldReleaseObj := oldObj.(*helmCrdV1.HelmRelease)
+				if releaseObjChanged(oldReleaseObj, newReleaseObj) {
+					queue.Add(key)
+				}
```

Comment:
Add something like `else log.Debugf("Ignoring update event on unchanged object %v", newReleaseObj)` here so we can debug it when we find out we're ignoring something that was important.